src/nodes/chunk_dispatch/chunk_insert_state.c

```diff
@@ -718,6 +715,14 @@ ts_chunk_insert_state_create(const Chunk *chunk, ChunkDispatch *dispatch)
 	return state;
 }
 
+void
+set_compression_status(ChunkInsertState *state, const Chunk *chunk)
```

Comment:
```suggestion
ts_set_compression_status(ChunkInsertState *state, const Chunk *chunk)
```
Non-static functions should have ts_ prefix